package.json

```diff
@@ -44,7 +44,7 @@
     "prettier": "prettier . --write",
     "lint:fix": "eslint . --fix",
     "lint": "prettier . --check && eslint .",
-    "postinstall": "opencollective-postinstall",
+    "postinstall": "opencollective-postinstall && sh build/husky.sh",
```

Comment:
Could we not run `npx husky install` as part of `postinstall` and remove the `build/husky.sh` script altogether?

 ```json
"postinstall": "opencollective-postinstall && npx husky install",
```

If do go this route, we would lose our own custom console output ("Husky installing ..."), but I'm guessing Husky displays its own install messages in the console, no?